### PR TITLE
Update phi3-v.md with DirectML

### PR DESCRIPTION
**Pull request by natke**
Add DirectML instructions to Phi-3 vision tutorial

### CODE REVIEW

File: docs/genai/tutorials/phi3-v.md [hunk 1]
```diff
@@ -91,6 +88,7 @@ Support for Windows machines with GPUs other than NVIDIA is coming soon!
 
    ```bash
    curl https://raw.githubusercontent.com/microsoft/onnxruntime-genai/main/examples/python/phi3v.py -o phi3v.py
+   pip install pyreadline3
```
Comment (kunal-vaishnavi) [inline docs/genai/tutorials/phi3-v.md:91]: Is `pyreadline3` required for the tutorial?

File: docs/genai/tutorials/phi3-v.md [hunk 2]
```diff
@@ -81,7 +79,6 @@ Support for Windows machines with GPUs other than NVIDIA is coming soon!
    * CUDA 12
 
    ```bash
-   pip install numpy
    pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
```
Comment (kunal-vaishnavi) [inline docs/genai/tutorials/phi3-v.md:82]: Can `--pre` be removed now that there are published stable releases that support Phi-3 vision?

File: docs/genai/tutorials/phi3-v.md (hunk 1 above)
Comment (kunal-vaishnavi) [inline docs/genai/tutorials/phi3-v.md:91]: Why is `pyreadline3` required for the tutorial?

File: docs/genai/tutorials/phi3-v.md [hunk 3]
```diff
@@ -152,3 +150,42 @@ Support for Windows machines with GPUs other than NVIDIA is coming soon!
    The products include Chocolade, Gummibarchen, Scottish Longbreads, Sir Rodney's Scones, Tarte au sucre,
    and Chocolate Biscuits. The Grand Total column sums up the sales for each product across the two quarters.</s>
    ```
+
+## Run with DirectML
```
Comment (kunal-vaishnavi) [inline docs/genai/tutorials/phi3-v.md:154]: Should we re-order the examples to first show DML, then CUDA, and finally CPU to match the recommended order in the [model cards](https://huggingface.co/microsoft/Phi-3-vision-128k-instruct-onnx-directml#onnx-models)?

File: docs/genai/tutorials/phi3-v.md (hunk 1 above)
Comment (natke) [reply docs/genai/tutorials/phi3-v.md:91]: It's used to read the image file location from the command line

File: docs/genai/tutorials/phi3-v.md (hunk 3 above)
Comment (natke) [reply docs/genai/tutorials/phi3-v.md:154]: Yes, we could do that. I played with the order

File: docs/genai/tutorials/phi3-v.md (hunk 2 above)
Comment (natke) [reply docs/genai/tutorials/phi3-v.md:82]: Fixed

File: docs/genai/tutorials/phi3-v.md (hunk 3 above)
Comment (natke) [reply docs/genai/tutorials/phi3-v.md:154]: Actually I'll finish this PR with the original order to keep it simple then submit a new one with a simple re-order

**Review by kunal-vaishnavi — APPROVED** [review]

**Pull request merged**